src/index.html

```diff
@@ -6,13 +6,182 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge" />
     <title>Weather App</title>
     <link rel="stylesheet" href="main.bundle.css" />
+    <style>
+      .container {
```

Comment:
uwaga na temat styli które trzeba przenieśc do odpowiednich plików, 
jeżeli korzystamy z metody BEM, umożliwia nam to swobodne zagnieżdzanie styli, trzeba pamiętać, żeby nie popłynąć i starać się zagnieżdzać maksymalnie do 3 poziomu czyli do poziomu modyfikatoru, w BEM mamy 3 poziomy
- bloki
- elementy
- modyfikatory

Dodatkowo trzeba zdefiniować zmienne, dzięki czemu w przyszłości jak będziemy chcieli coś zmienić to zmieniamy w jednym miejscu w pliku styli variables i cała strona nam się aktualizuje.
Wyobraż sobie, że masz bardzo duży projekt gdzie w kilkuset miejscach dodajesz ręcznie kolory hexem lub czcionki 
i klient przychodzi, że wszędzie zmieniamy kolor, wtedy masz bardzo męczące zadanie do zrobienia, warto w tym momencie stworzyć plik variables i tam zdefiniować kika rzeczy takich jak:
- kolory
- rozdzielczości
- rozmiar czcionek 
- czcionki 
- czy nawet jak nasza strona, jest w stylistyce zaokrąglonych elementów to zdefiniować coś takiego jak border-radius dla elementów.


artykuł SASS:
https://bedekodzic.pl/sass-p1/ 
artykuł BEM:
https://www.nafrontendzie.pl/metodyki-css-2-bem